### ISSUE TITLE
Message not received by non textsecure recipients

### ISSUE THREAD

**Issue by jjanicki02** [WhisperSystems/TextSecure#2399]
I just bought a note 4 on Verizon. I downloaded textsecure (which works fine on my old gs3) and made it the default messenger.  Non textsecure recipients are not receiving my outgoing sms messages. Messages from non textsecure users are received.  Messages to/from textsecure user are successful.  I tried sending an outgoing message to an email and that worked. Please let me know if there is a way to resolve these issues or if more info is necessary.

**Comment by jjanicki02** [comment]
FYI - it shows the message as sent

**Comment by agrajaghh** [comment]
Did you try it with different non textsecure users? Can you post a debug log after sending a sms to a non textsecure user?

**Comment by SmalleyJ** [comment]
I'm having the same issue but I've noticed it's only certain iPhone users I'm having the problem with.

**Comment by jjanicki02** [comment]
For me the problem seemed to have resolved itself.   Now that you say that, the two non-textsecure users that I originally tested sending messages to (and who did not receive the messages) were iphone users.  I will leave this open in case you want to pursue it further.

**Comment by SmalleyJ** [comment]
I appreciate that. I will test it more today and submit a debug log if the problem continues.

**Comment by SmalleyJ** [comment]
Just tested it and confirmed it is still an issue. Debug log below: https://gist.github.com/f69297d61f57c3ceffb9

**Comment by agrajaghh** [comment]
The "real" sending is done by the OS, and it reported that the message was sent successful (return code -1)
```
W/SmsSendJob(14523): Sending message: 2270
W/SmsSentJob(14523): Got SMS callback: org.thoughtcrime.securesms.SendReceiveService.SENT_SMS_ACTION , -1 
```

**Comment by moxie0** [comment]
We have no control over this.  We hand the SMS to the OS, and it handles communication with the radio.  This is either a problem with your OS or with your mobile service.

**Comment by SmalleyJ** [comment]
Ok, thank you for the quick response.

**Comment by jjanicki02** [comment]
SmalleyJ - FYI the only things I did after noticing the problem was to uninstall, reinstall text secure, rebooted the phone, and since I am on verizon, I enabled manual MMS settings and changed the MMSC URL to http://mms.vtext.com/servlets/mms.  On my old phone I had to make that change for MMS to come through.  I tested it immediately after i did that and it still wasn't working.  The next day I tried one more time and it was working. I'm not sure if any of this helped to make it work on my phone but I thought I would throw that out there.
